### DISCUSSION TITLE
Intermittent hot reload

### DISCUSSION THREAD

**Discussion by calpage18** [vercel/next.js#14134] [Help]
Hi all,

When running in dev mode, the page will try to reload intermittently, even when I've not interacted with the page or the code. It then also fails to perform a fast refresh, with the message `[Fast Refresh] performing full reload` and does a full reload, losing all the data entered into forms so far.  

It also happens occasionally when clicking a button that performs a `router.push()`, but not every time, usually only the first time.

Is there an obvious reason it would be performing these intermittent reloads? And if so, would `_app.js` being a class component be the reason for the full reload instead of fast refresh?

**Comment by calpage18** [comment]
If anyone can answer these questions, it’d be much appreciated. Thanks!

**Comment by rafaelalmeidatk** [comment]
Do you have any components without a name (declared with anonymous functions, HOCs included)? This is one of the reasons Fast Refresh won't work

**Comment by calpage18** [comment]
I don’t have any anonymous function components, and the only HoC I have is _app.js being exported wrapped in withApollo as per the old example in the repo. I notice that the example has been updated to no longer use withApollo, so I converted mine to match, but the hard reload is still occurring.

Is it normal for fast refresh to intermittently occur without code edits? I guess it would be useful to know if I’m trying to debug why it’s trying to fast refresh in the first place or why it’s doing a full reload. 

Thanks

**Reply by calpage18** [reply]
So it turns out that some custom webpack overrides in next.config.js were to blame! Thanks for your help anyway @rafaelalmeidatk !

**Reply by rafaelalmeidatk** [reply]
Oh, great you solved it!

**Reply by devth** [reply]
@calpage18 can you share what webpack config was causing it? I'm seeing the same problem.

**Reply by JeffLuong** [reply]
@calpage18 Bumping on this, can you share what the webpack config that was causing this? I'm also experiencing the same issue.

For context, I'm using Next `10.2.3` with webpack 5
